velox/exec/tests/SpillTest.cpp

```diff
@@ -42,12 +42,21 @@ class SpillTest : public testing::Test,
       int64_t targetFileSize,
       int numPartitions,
       int numBatches,
+      int numDuplicates,
       int64_t expectedNumSpilledFiles) {
     auto tempDirectory = exec::test::TempDirectoryPath::create();
     const std::string kSpillPath = tempDirectory->path + "/test";
     std::shared_ptr<FileSystem> fs;
     std::vector<std::string> spilledFiles;
     const int kNumRowsPerBatch = 100'000;
+    std::vector<std::vector<int64_t>> values(numBatches);
+    std::vector<int64_t> expectedValues(kNumRowsPerBatch * numBatches * 2);
+    for (int i = 0, value = -10; i < kNumRowsPerBatch * numBatches * 2; ++value) {
```

Comment:
It is a bit hard for me to understand the dataset being created here. Would you explain what the data looks like?

Reply:
Updated. Thanks for pointing out!